augauto_module_path: "/home/demo/Workspace/seung/AugmentedAutoencoder"
exp_group: "exp_group/Ikea_stefan_middle"
gpu_id: "1"
width: 640
height: 480
is_thresh: 0.5
class_id: 3
crop_offset: 0.1
